### PR TITLE
Added e2e tests for `post.scheduled` event

### PR DESCRIPTION
**Pull request by jennyxchang**
refs #15537
Added e2e test for post.scheduled event to increase webhook test coverage

### CODE REVIEW

File: ghost/core/test/e2e-webhooks/posts.test.js [hunk 1]
```diff
@@ -154,4 +154,57 @@ describe('post.* events', function () {
...
+                }]
+            })
+            .expectStatus(201);
+
+        const id = res.body.posts[0].id;
+        const scheduledPost = res.body.posts[0];
+        scheduledPost.status = 'scheduled';
+        const publishedTime = new Date();
+        publishedTime.setDate(publishedTime.getDate() + 1);
+        scheduledPost.published_at = publishedTime.toISOString();
```
Comment (naz) [inline ghost/core/test/e2e-webhooks/posts.test.js:181]: For date calculations in the codebase there's a preference to use a date library - [like here](https://github.com/TryGhost/Ghost/blob/34439f2e28e4603e9ef30c86954099d56a7c9ef2/ghost/core/test/regression/api/admin/schedules.test.js#L36).

The following: 
```js
moment().add(30, 'seconds').toDate()
```

is easier to read and later migrate to a different library (we have been trying to [migrate away to Luxon from momentjs](https://github.com/TryGhost/Ghost/issues/13648)).

**Review by naz — CHANGES_REQUESTED** [review]
Just a minor change request to use a library instead of manual date calculation. The rest looks 👌👌👌 Thanks @jennyxchang!

File: ghost/core/test/e2e-webhooks/posts.test.js (hunk 1 above)
Comment (jennyxchang) [reply ghost/core/test/e2e-webhooks/posts.test.js]: Thanks for the suggestion! I updated the date calculation using the date library.

**Comment by jennyxchang** [comment]
Thanks for reviewing! I updated the date calculation as requested.

**Review by naz — APPROVED** [review]
These changes are good and ready to be merged. Thank you @jennyxchang 🙌

**Comment by ErisDS** [comment]
Hi @jennyxchang  thanks so much for this PR. Sorry for the delay, it has now been merged 🎉 and will appear in the next release of Ghost - usually Fridays.

I'm not sure if you found this through hacktoberfest, but I've added the accepted label to this PR to make sure it counts. We have plenty of open issues and ongoing projects for hacktoberfest and beyond. We would love to see you around again 👋

**Pull request merged**